src/app/pages/listings/contents/listings-content/listings-content.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { map, tap } from 'rxjs/operators';
+import { ListingsStore } from '../service/listings-store';
+
+@Component({
+  selector: 'app-listings-content',
+  templateUrl: './listings-content.component.html',
+  styleUrls: ['./listings-content.component.scss']
+})
+export class ListingsContentComponent implements OnInit {
+  hasPagination = false;
+  listingStore$ = this.listingsStore.listings$.pipe(tap(listing=>this.hasPagination = listing?.length === 75));
```

Comment:
this.hasPagination = listing?.length >= 75 

I think if it is greater than or equal to 75 then we show load more

Reply:
Can I fix this while doing pagination implementation?

Reply:
Yes